### PR TITLE
Update guzzlehttp/guzzle to 5.3.0

### PR DESCRIPTION
**Pull request by rullzer**
Also update depenencie react/promise to 2.2.1

### CODE REVIEW

File: react/promise/README.md [hunk 1]
```diff
@@ -0,0 +1,823 @@
+React/Promise
```
Comment (DeepDiver1975) [inline react/promise/README.md]: @rullzer I guess we can gitignore this file

File: react/promise/composer.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (DeepDiver1975) [inline react/promise/composer.json]: gitignore as well

**Comment by rullzer** [comment]
@DeepDiver1975 fixed. Yeah you are right. We are not really consistent... with some they are there with soem not.

**Comment by DeepDiver1975** [comment]
THX :+1:

**Pull request merged**